test/e2e/checkpoint_test.go

```diff
@@ -30,6 +30,11 @@ var _ = Describe("Podman checkpoint", func() {
 		if !criu.CheckForCriu() {
 			Skip("CRIU is missing or too old.")
 		}
+		hostInfo := podmanTest.Host
+		if hostInfo.Distribution == "fedora" {
+			Skip("Checkpoint tests appear to fail on F29.")
+		}
+
```

Comment:
I'll let @mheon make the call, but I'd suggest a TODO comment here as to when this should be turned back on.

Reply:
Agree with a TODO

Reply:
I would also suggest adding a BZ or issue reference to the comment.  Just in case somebody wonders 'why' (in detail)

Reply:
Updated skipping message and added a comment including a link to the ticket.